tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -454,12 +454,13 @@
 
         /**
          * The plugin name of the currently selected dashboard, or `null` if no
-         * dashboard is selected. A `null` value here is represented by
-         * an empty string in the hash, and vice versa.
+         * dashboard is selected, which corresponds to an empty hash. Defaults
+         * to the value stored in the hash.
          */
         _selectedDashboard: {
           type: String,
-          value: null,
+          value: getString(TAB, /*useLocalStorage=*/false) || null,
+          observer: '_selectedDashboardChanged'
```

Comment:
Maybe place this alongside other observers so others can easily skim through all observers.

Reply:
Hm, but there are other single-property observers defined inline above for _title and _router.  Do you think they should all be moved to the _observers field?  I had looked around the codebase a little and it seemed like observers were declared inline pretty often.

Also, it's not really significant here, but apparently there is a semantic difference - inline produces a "simple observer" whereas declaration in observers produces a "complex observer" (https://www.polymer-project.org/1.0/docs/devguide/observers), though for single-property observers that aren't using any of the complex features there's basically no difference (only that the simple version also receives the old value as a second argument, but we aren't using that here, so it could be declared either way).